locales/en/translation.json

```
@@ -0,0 +1,93 @@
+{
+  "$Language": {
+    "en": "English",
+    "pt-BR": "Português (Brasil)"
+  },
+  "$Preferences": {
+    "title": "Time to Leave - Preferences",
+    "userPreferences": "User preferences",
+    "workingDays": "Working days",
+    "sun": "Sun",
+    "mon": "Mon",
+    "tue": "Tue",
+    "wed": "Wed",
+    "thu": "Thu",
+    "fri": "Fri",
+    "sat": "Sat",
+    "hideNonWorkingDay": "Hide non-working days (Month View)",
+    "hoursPerDay": "Hours per day",
+    "appBehavior": "App Behavior",
+    "notification": "Notification",
+    "allowRecurringNotifications": "Allow recurring notifications",
+    "minutesBetweenNotifications": "Minutes between notifications",
+    "startOnLogin": "Start on Login",
+    "countTodayOnTotals": "Count Today in Totals",
+    "overallBalanceStart": "Overall Balance Start Date",
+    "minimizeToTray": "Minimize button should minimize to tray",
+    "closeMinimizeToTray": "Close button should minimize to tray",
+    "themes": "Themes",
+    "systemDefault": "System Default",
+    "light": "Light",
+    "dark": "Dark",
+    "cadentStar": "Cadent Star",
+    "view": "View",
+    "month": "Month",
+    "day": "Day",
+    "numberOfEntries": "Number of Entries",
+    "fixed": "Fixed",
+    "flexible": "Flexible",
+    "language": "Language"
+  },
+  "$Menu": {
+    "about": "About",
+    "all-clear": "All cleared!",
+    "all-files": "All Files",
+    "cancel": "Cancel",
+    "check-for-updates": "Check for updates",
+    "clear-database": "Clear Database",
+    "confirm-clear-all-data": "Are you sure you want to clear all the data?",
+    "confirm-import-db": "Are you sure you want to import a database? It will override any current information.",
+    "confirm-migrate-to-flexible": "Are you sure you want to migrate the fixed calendar database to the flexible calendar?\n\nThe existing flexible calendar database will be cleared.",
+    "copy": "Copy",
+    "could-not-be-loaded": "could not be loaded",
+    "Cut": "Cut",
+    "database-export": "Database export",
+    "database-imported": "Database imported",
+    "database-migrated": "Database migrated",
+    "database-was-exported": "Okay, database was exported.",
+    "edit": "Edit",
+    "exit": "Exit",
+    "export-database": "Export database",
+    "export-db-to-file": "Export DB to file",
```

```suggestion
    "export-db-to-file": "Export database to file",
```
I think it's better if we stop shortening it here, it will also potentially be easier for translators